src/main.js

```diff
@@ -60,6 +62,8 @@ function init(config) {
         }
     } else if (isCommit()) {
         codeReviewFeatures(config);
+    } else if (isComparing()) {
+        mergeCompare(config);
```

Comment:
Do you need to pass this `config` object ?

Reply:
Nope, i'll remove it